dev/examples.el

```diff
@@ -733,7 +733,26 @@ new list."
   (defexamples -->
     (--> "def" (concat "abc" it "ghi")) => "abcdefghi"
     (--> "def" (concat "abc" it "ghi") (upcase it)) => "ABCDEFGHI"
-    (--> "def" (concat "abc" it "ghi") upcase) => "ABCDEFGHI"))
+    (--> "def" (concat "abc" it "ghi") upcase) => "ABCDEFGHI")
+
+  (defexamples -some->
+    (-some-> '(2 3 5)) => '(2 3 5)
+    (-some-> 5 square) => 25
+    (-some-> nil square) => nil
+    (-some-> 5 even? square) => nil)
```

Comment:
Only the first three examples are passed to the readme. Could you move this one up, or add some more when we see how the nil is handled? Because with just one form it's rather boring :)